config.json

```diff
@@ -1033,7 +1033,12 @@
       "core": false,
       "difficulty": 9,
       "slug": "pythagorean-triplet",
-      "topics": null,
+      "topics": [
+       "integer",
+	"lists",
+	"logic",
+	"mathematics"
```

Comment:
I'm afraid the indentation is still not quite right. `lists`, `logic` and `mathematics` all need to be indented two spaces more than `topics`, they currently appear to be indented quite a few spaces less than `topics`. `integer` needs to be indented two spaces in from `topics`, it's currently indented one space in from `topics`. You say it looked fine in the "Changed files" tab. Did you look at all changes? It doesn't look fine when I look at it...

Sorry to be a bit picky about this but it is important to get the spacing right, otherwise the file becomes harder to read and maintain :)

Strange, I'm using Intellij on Windows and I haven't seen any issues where spacing appears correct in Intellij but then isn't when pushed to Github. Any idea what's going on @exercism/java?

Reply:
No idea unless it's a tabs vs spaces thing. We can merge as-is and it will be fixed automatically the next time we (the maintainers) run `configlet fmt`.

Reply:
Sure :)